staging/src/k8s.io/apiextensions-apiserver/go.mod

```diff
@@ -35,13 +35,27 @@ require (
 )
 
 replace (
+	github.com/fatih/color => github.com/fatih/color v1.6.0
+	github.com/golang/protobuf => github.com/golang/protobuf v1.3.1
+	github.com/google/btree => github.com/google/btree v0.0.0-20180813153112-4030bb1f1f0c
+	github.com/grpc-ecosystem/go-grpc-middleware => github.com/grpc-ecosystem/go-grpc-middleware v0.0.0-20190222133341-cfaf5686ec79
+	github.com/grpc-ecosystem/grpc-gateway => github.com/grpc-ecosystem/grpc-gateway v1.3.0
+	github.com/mattn/go-isatty => github.com/mattn/go-isatty v0.0.3
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.4
+	github.com/soheilhy/cmux => github.com/soheilhy/cmux v0.1.3
+	github.com/xiang90/probing => github.com/xiang90/probing v0.0.0-20160813154853-07dd2e8dfe18
+	go.uber.org/atomic => go.uber.org/atomic v0.0.0-20181018215023-8dc6146f7569
+	go.uber.org/multierr => go.uber.org/multierr v0.0.0-20180122172545-ddea229ff1df
+	go.uber.org/zap => go.uber.org/zap v0.0.0-20180814183419-67bc79d13d15
 	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20181025213731-e84da0312774
 	golang.org/x/lint => golang.org/x/lint v0.0.0-20181217174547-8f45f776aaf1
+	golang.org/x/net => golang.org/x/net v0.0.0-20190812203447-cdfb69ac37fc
 	golang.org/x/oauth2 => golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a
 	golang.org/x/sync => golang.org/x/sync v0.0.0-20181108010431-42b317875d0f
 	golang.org/x/sys => golang.org/x/sys v0.0.0-20190209173611-3b5209105503
 	golang.org/x/text => golang.org/x/text v0.3.1-0.20181227161524-e6919f6577db
 	golang.org/x/time => golang.org/x/time v0.0.0-20161028155119-f51c12702a4d
+	google.golang.org/grpc => google.golang.org/grpc v1.23.0
```

Comment:
`v1.23.1` ? I see some updates to `v1.23.1`.  There are a bunch of other `v1.23.0` usages below that all need to be updated too.

Reply:
Yes, I just updated the lint-dependencies, the first push was done before that. Now it's v1.23.1.